nflow-engine/src/main/resources/scripts/db/update-5.5.0-x/h2.update.ddl.sql

```diff
@@ -0,0 +1 @@
+alter table nflow_workflow add started timestamp;
```

Comment:
Is it possible to have UPDATE commands that populate `started` field based on data in nflow_action table?

Reply:
Added